camera/camera_balsaq.hh

```diff
@@ -46,13 +37,12 @@ class CameraBq : public BalsaQ {
   // The camera capture times are reported in millseconds from monotonic clock start
   // This function reconstructs a a reasonable approximation of the *vehicle* time
   // associated with that monotonic time
-  Timestamp msec_monotonic_to_vehicle_monotonic(long int msec_monotonic) const;
   static const int CAMERA_FPS = 10;
   PublisherPtr publisher_;
   cv::VideoCapture cap_;
   Timestamp last_msg_recvd_timestamp_;
 
-  std::string camera_serial_number_;
+  Camera camera;
```

Comment:
- Member variables should be suffixed with `_`
- Possibly no need to store this at all